src/chat-bubble/interfaces.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export interface ChatBubbleProps {
+  /** Avatar slot paired with the chat bubble. */
+  avatar: React.ReactNode;
+
+  /** Defines the type of the chat bubble and sets its color accordingly.  */
+  type: ChatBubbleProps.Type;
+
+  /** Content of the chat bubble */
+  children: React.ReactNode;
+
+  /** Actions slot of the chat bubble, placed at the bottom left. */
```

Comment:
nit: it is not in the bottom left when direction is RTL.

Reply:
oh right I'll just say footer